WcaOnRails/spec/requests/api_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "API mics" do
```

Comment:
nit: What does "API mics" mean?

Reply:
Certainly a typo, should've been "API misc", as the main controller has a bunch of different stuff in it.